podvm/files/etc/systemd/system/go.mod

```diff
@@ -0,0 +1,3 @@
+// Empty go.mod file to exclude the run-kata\x2dcontainers.mount file
+// from the cloud-api-adaptor Go module when cloud-api-adaptor is
+// imported by other Go projects
```

Comment:
Does it make sense to exclude all the files under the `podvm` directory instead of only the systemd files?

Reply:
Thanks @bpradipt. Good suggestion and I moved to `go.mod` files to under the `podvm` and azure` directories.